class/Traits/Database.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Trait for database operations.
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Traits;
+
+use BracketSpace\Notification\Utils\Cache\ObjectCache;
+
+/**
+ * Database trait
+ */
+trait Database {
+
+	/**
+	 * Get all prepared users directly from database.
+	 *
+	 * @return array $results user results.
+	 */
+	public function get_all_users_from_db() {
```

Comment:
Redundancy in name, better:

```suggestion
	public function get_all_users() {
```